nrich-notification/README.md

```diff
@@ -263,7 +263,11 @@ In [`DefaultNotificationResolverService`][default-notification-resolver-service-
 
 ### Custom exception notification data
 
-Depending on the context of notification creation, if a notification is created while resolving an exception, then the action code is a **fully qualified class name for that exception**.
+Depending on the context of notification creation, if a notification is created while resolving an exception, then the default action code is a **fully qualified class name for that exception**.
+If an exception implements [`ExceptionWithMessageCode`][exception-with-message-code-url] then notification content is resolved through provided message code
+and if an exception implements [`ExceptionWithMessage`][exception-with-message-url] then notification content is equal to exceptions message
+(title is still resolved from fully qualified class name in both cases). If additional arguments are required when resolving message for provided exception exception should implement
```

Comment:
` If additional arguments are required when resolving message, exception should implement...` is clear enough here.